src/hotspot/share/prims/jvm.cpp

```diff
@@ -378,7 +378,7 @@ JVM_ENTRY(jobjectArray, JVM_GetProperties(JNIEnv *env))
   // Add the sun.management.compiler property for the compiler's name
   {
 #undef CSIZE
-#if defined(_LP64) || defined(_WIN64)
+#if defined(_LP64)
```

Comment:
Windows is actually LLP64 programming model not LP64. Does Windows x64 define _LP64 or is that something we do in our build?

Reply:
It's something we do in our build

Reply:
It seems like the `_WIN64` check here was never useful.  It's also been there since before the
mercurial age.

Reply:
The "mercurial age". Sounds like something in-between the stone age and the bronze age. :-D